core/themes/basis/template.php

```diff
@@ -53,6 +67,13 @@ function basis_preprocess_page(&$variables) {
   }
 }
 
+/**
+ * Every time we add supplemental CSS we add the version number here.
+ */
+function basis_supplemental_css_versions() {
+  return ['1.23.0', '1.28.4', '1.30.0','1.31.5'];
```

Comment:
As much as I like the simplified array syntax, it's not yet our coding standard.
```suggestion
  return array('1.23.0', '1.28.4', '1.30.0','1.31.5');
```

Reply:
Sounds good. This was not a conscious choice on my part. I was influenced by my AI assistant and lack of knowing better. ;-)